src/SFA.DAS.EmployerIncentives/Interfaces/ILegalEntitiesService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using SFA.DAS.EmployerIncentives.InnerApi.Requests;
+using SFA.DAS.EmployerIncentives.Models;
+
+namespace SFA.DAS.EmployerIncentives.Interfaces
+{
+    public interface ILegalEntitiesService
+    {
+        Task<AccountLegalEntity[]> GetAccountLegalEntities(long accountId);
+        Task<AccountLegalEntity> GetLegalEntity(long accountId, long accountLegalEntityId);
+        Task DeleteAccountLegalEntity(long accountId, long accountLegalEntityId);
+        Task CreateLegalEntity(long accountId, AccountLegalEntityCreateRequest accountLegalEntity);
```

Comment:
Can't the accountId go in the request object?

Reply:
Done